Ginger/GingerCoreNETUnitTest/GingerRunnerTests/GingerRunnerTest.cs

```diff
@@ -265,7 +285,83 @@ public void TestRunsetConfigBFVariables()
 
         }
 
-        
+
+        [TestMethod]
+        public void TestDyanamicRunsetOprations()
+        {
+            RunSetConfig runSetConfig = CreteRunsetWithOperations();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename 'runSetConfig' which hides the field with the same name.](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4629760)